### PR TITLE
[minio] Update set/getBucketPolicy definitions to 5.0

### PR DESCRIPTION
**Pull request by doomsower**
This PR updates `getBucketPolicy`, `setBucketPolicy` and `bucketExists` definitions from v 4.0 to v 5.0

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://docs.minio.io/docs/javascript-client-api-reference#getBucketPolicy
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/minio/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for minio 4.0
+// Type definitions for minio 5.0
 // Project: https://github.com/minio/minio-js#readme
 // Definitions by: Barin Britva <https://github.com/barinbritva>
```
Comment (barinbritva) [inline types/minio/index.d.ts]: Need to add author :)

File: types/minio/minio-tests.ts [hunk 2]
```diff
@@ -106,8 +106,14 @@ minio.removeAllBucketNotification('testBucket');
...
 
-minio.getBucketPolicy('testBucket', 'pref_', (error: Error|null, policy: Minio.PolicyValue) => { console.log(error, policy); });
-minio.getBucketPolicy('testBucket', '');
-
-minio.setBucketPolicy('testBucket', '', Minio.Policy.READWRITE, (error: Error|null) => { console.log(error); });
-minio.setBucketPolicy('testBucket', 'pref_', Minio.Policy.WRITEONLY);
+minio.getBucketPolicy('testBucket', (error: Error|null, policy: string) => { console.log(error, policy); });
+minio.getBucketPolicy('testBucket');
+
+const testPolicy = `{"Version":"2012-10-17","Statement":[{"Action":["s3:GetBucketLocation"],"Effect":"Allow",
```
Comment (barinbritva) [inline types/minio/minio-tests.ts]: How can I get string like this? Is it result of JSON.stringify from newPostPolicy?

**Review by barinbritva — APPROVED** [review]

File: types/minio/minio-tests.ts (hunk 2 above)
Comment (doomsower) [reply types/minio/minio-tests.ts]: Copied from minio-js [tests](https://github.com/minio/minio-js/blob/6c752fe370a49ac2efedc9b41e5e7e4591088772/src/test/functional/functional-tests.js#L631)

File: types/minio/index.d.ts (hunk 1 above)
Comment (doomsower) [reply types/minio/index.d.ts]: I believe in this case I'll receive messages from definitely typed bot asking me to review other PRs, and I don't feel like I'm entitled to do that

**Pull request merged**